packages/@tailwindcss-upgrade/src/migrate-js-config.ts

```diff
@@ -115,6 +125,15 @@ async function migrateTheme(unresolvedConfig: Config & { theme: any }): Promise<
     css += `  --${keyPathToCssProperty(key)}: ${value};\n`
   }
 
+  if ('keyframes' in themeValues) {
+    containsThemeKeys = true
+    css += '\n' + keyframesToCss(themeValues.keyframes)
+  }
+  if ('extends' in themeValues && 'keyframes' in themeValues.extends) {
```

Comment:
because you're doing that deepMerge isn't this block unnecessary? Might be missing something tho…

Reply:
OH yes you're absolutely right I forgot I added the deep merge in a previous PR even though this PR has a diff for it too Loool